packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -1134,6 +1111,31 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with Restoratio
     return _hasDecoration ? TextAlignVertical.center : TextAlignVertical.top;
   }
 
+  void _onClearButtonTapped() {
+    // Special handle onChanged for ClearButton
```

Comment:
consider consolidating this comment into one sentence. maybe something like `Handler for ClearButton that calls onChanged when the clear button is tapped`.

Reply:
I basically moved the code and the comment to an instance method. I'll update the comment.